pkg/kubelet/dockertools/manager.go

```diff
@@ -1543,7 +1545,7 @@ func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubeletTypes.Doc
 		return "", nil, err
 	}
 
-	id, dockerContainer, err := dm.runContainerInPod(pod, container, netNamespace, "")
```

Comment:
this should only use `dockerHostNamespace` as the IPC namespace if `pod.Spec.HostIPC` is true, right?